traits/ctraits.c

```diff
@@ -491,7 +491,7 @@ set_delete_property_error(has_traits_object *obj, PyObject *name)
 static void
 unknown_attribute_error(has_traits_object *obj, PyObject *name)
 {
-    return PyErr_Format(
```

Comment:
should this function become a `static int` instead of a `static void` like the similar `invalid_attribute_error` function?

Reply:
I don't think that would buy us anything, except consistency. The `static int` on the other functions is a little odd, because the return value is mostly useless - it doesn't pass on any information. It's a minor convenience if you happen to be calling `invalid_attribute_error` from a function that expects to return an `int`, but I don't see a clear reason why that would always be the case - you might just as well be calling `invalid_attribute_error` from a function that expects to return a pointer instead.